source/guides/core-concepts/retry-ability.md

```diff
@@ -358,6 +358,76 @@ The above test retries getting the element and invoking the text of the element
 
 {% imgTag /img/guides/retry-ability/random-number-callback.gif "Random number using callback" %}
 
+## Use aliases
+
+When using {% url `cy.stub()` stub %} or {% url `cy.spy()` spy %} to test application's code, a good practice is to give it an alias and use the `cy.get('@alias').should('...')` assertion to retry.
+
+For example, when confirming that the button component invokes the passes `click` prop, the following test might or might not work:
+
+### {% fa fa-warning red %} Incorrectly checking if the stub was called
+
+```js
+const Clicker = ({ click }) => (
+  <div>
+    <button onClick={click}>Click me</button>
+  </div>
+)
+
+it('calls the click prop twice', () => {
+  const onClick = cy.stub()
+  // "mount" function comes from
+  // https://github.com/bahmutov/cypress-react-unit-test
+  mount(<Clicker click={onClick} />)
+  cy.get('button')
+    .click()
+    .click()
+    .then(() => {
+      // works in this case, but not recommended
+      // because https://on.cypress.io/then does not retry
+      expect(onClick).to.be.calledTwice
+    })
+})
+```
+
+The above example will fail if the component calls the `click` prop after a delay.
+
+```js
+const Clicker = ({ click }) => (
+  <div>
+    <button onClick={() => setTimeout(click, 500)}>Click me</button>
+  </div>
+)
+```
+
+{% imgTag /img/guides/retry-ability/delay-click.png "Expect fails the test without waiting for the delayed stub" %}
+
+The test finishes before the component calls the `click` prop twice, and without retrying the assertion `expect(onClick).to.be.calledTwice`.
+
+### {% fa fa-check-circle green %} Correctly waiting for the stub to be called
+
+We recommend to give the stub an alias using {% url `.as` as %} command and use `cy.get('@alias').should(...)` assertions.
+
+```js
+it('calls the click prop', () => {
+  const onClick = cy.stub().as('clicker')
+  // "mount" function comes from
+  // https://github.com/bahmutov/cypress-react-unit-test
+  mount(<Clicker click={onClick} />)
+  cy.get('button')
+    .click()
+    .click()
+
+  // good practice 💡
+  // auto-retry the stub until it was called twice
+  cy.get('@clicker').should('have.been.calledTwice')
+})
+```
+
+{% imgTag /img/guides/retry-ability/click-twice.gif "Retrying the assertions using a stub alias" %}
+
+Watch the short video below to see this example in action
+{% video YouTube AlltFcsIFvc %}
```

Comment:
Miscapitalized youtube here. Try to run the docs locally when adding larger blocks to catch this. 

<img width="568" alt="Screen Shot 2020-06-02 at 11 51 37 AM" src="https://user-images.githubusercontent.com/1271364/83482924-76fd9980-a4c7-11ea-9abd-47d785655f15.png">

Reply:
Actually I bet our text linter auto-fixed this, cause you have to testlint ignore these 🙄